### ISSUE TITLE
[REQ] Option to send message only to users with VIP

### ISSUE THREAD

**Issue by crashzk** [wiruwiru/AutomaticAds-CS2#1]
I believe that using CSSharp's own flag system could be perfect, so we can configure the message to be sent only to those who have a certain flag.

**_Note: From what I've seen, support for multiple messages is already in the roadmap, it would be interesting if each message in the future had this flag option._**

**Comment by wiruwiru** [comment]
That's a good idea, I'll add it to the TODO list.

**Comment by crashzk** [comment]
Just a note, in addition to the flag system, it would also be interesting to have the option of sending messages on a specific map only.

Example:
```
"AD1"
{
   "en": "Message AD1 only VIP",
   "flag": "@css/vip",
   "map": "de_dust2"
},

"AD2"
{
   "en": "Message AD2",
   "map": "de_mirage"
}
```
Then AD2 would only be sent if the map is `de_mirage`.

**Comment by wiruwiru** [comment]
Yes, it is a good implementation, I will try to implement it in a few days. 

I will use `“map”: “all”` for those announcements that can be sent on all maps.

**Comment by wiruwiru** [comment]
Added in [AutomaticAds v1.0.2](https://github.com/wiruwiru/AutomaticAds-CS2/releases/tag/build-11)
